### PR TITLE
WMS #11276:Get Started with Oracle Integration 3 (OCW folder update)

### PR DESCRIPTION
**Pull request by kkatta11**
# 11276

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [Yes ] All filenames are lower case (including folders, images, files, etc.)
- [ Yes] Filenames are descriptive
- [ Yes] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ Yes] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ Yes] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by haj — APPROVED** [review]
Looks good to me.

File: oic-gen3/cookbooks/erp-cloud-b2b/create-integration-flow/create-integration-flow.md
Comment (ramumg) [inline oic-gen3/cookbooks/erp-cloud-b2b/create-integration-flow/create-integration-flow.md:1]: Please change Estimated Time to Estimated Workshop Time

File: oic-gen3/cookbooks/erp-cloud-b2b/introduction/introduction.md
Comment (ramumg) [inline oic-gen3/cookbooks/erp-cloud-b2b/introduction/introduction.md:1]: Please change estimated time to Estimated Workshop Time

**Review by ramumg — APPROVED** [review]

File: oic-gen3/cookbooks/erp-cloud-b2b/introduction/introduction.md
Comment (kkatta11) [reply oic-gen3/cookbooks/erp-cloud-b2b/introduction/introduction.md:1]: @ramumg The QA doc itself says "Estimated Time". Did this change in the recent past?

**Comment by ramumg** [comment]
Hi @kkatta11 Please update your branch so we can go ahead and merge this PR.

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**